website/views/donations.py

```diff
@@ -119,8 +121,24 @@ def webapp_view_donation():
         log(f"Exception when getting a campaign for a donations: {e}", severity="ERROR")
         return render_template("errors/403.html"), 403
 
+    logs_url = None
+    try:
+        trace_id = request.args.get("trace_id")
+
+        project_req = requests.get(
+            "http://metadata.google.internal/computeMetadata/v1/project/project-id",
+            headers={"Metadata-Flavor": "Google"},
+        )
+        project_id = project_req.text
+
+        if trace_id and project_id:
+            logs_url = f"https://console.cloud.google.com/logs/query;query=trace%3D~%22projects%2F{project_id}%2Ftraces%2F{trace_id}%22?project={project_id}"
```

Comment:
Do we need to query limit to project when we're already operating within a particular project?

Reply:
Good call-out: TIL you can do [cross-project traces](https://cloud.google.com/trace/docs/cross-project-traces).

@pattishin do you mind if we drop the per-project filter?